newsletter/_posts/2018-01-25-CWP-Naples.md

```diff
@@ -0,0 +1,44 @@
+---
+title: CWP and Naples Workshop
+author: Graeme Stewart
+layout: newsletter
+---
+
+## CWP Roadmap
+
+The [Community White Paper](/activities/cwp.html){:target="_cwp"} process, that the HSF launched in 
+[San Diego](/2017/02/01/CWPWorkshopSanDiego.html){:target="_cwp_sandiego"} in January 2017 
+came to a successful conclusion
+at the end of last year, with the publication of *A Roadmap for HEP Software and 
+Computing R&D for the 2020s* to [arXiv](https://arxiv.org/abs/1712.06982){:target="_cwp_roadmap_arxiv"}.
+
+Motivated by the huge challenges of the next decade, particularly for the HL-LHC, the roadmap
+covers the whole of the HEP computing workflow, from event generation through simulation, reconstruction
+and analysis. The document also discusses our grid infrastructure components 
+and the vital glue of data management and security. For each area, work that the community
+should invest in to succeed in the future has been identified.
+
+The CWP process engaged hundreds of experts in the field and has helped to forge new links between
+people working in different experiments and in different regions who had not previously worked
+together. This is reflected in the degree of support we have in the community, with more that 270
+authors from more than 100 institutes. You can still sign the Roadmap - just send an email to 
+the [CWP Ghost Writers](mailto:hsf-cwp-ghost-writers@googlegroups.com?subject=CWP Signature) to lend
+your support to this important document.
+
+## HSF/WLCG Workshop
+
+Now that we have concluded the roadmap, the next thing to do is to start working on all the
+interesting projects that we have identified. To that end WLCG and the HSF are joining forces
+this Spring in Naples to have a 
+[joint workshop](https://indico.cern.ch/event/658060/overview){:target="_naples_workshop"}
+focused on our future software and computing development. 
```

Comment:
Just to be picky and show that I read ;-): I would put developmentS.

Reply:
Minor text nit: 
are joining forces "to hold a joint workshop" sounds a bit better, or perhaps "are co-sponsoring a workshop"
but otherwise it looks good.